scipy/stats/_multivariate.py

```diff
@@ -1578,6 +1611,128 @@ def rvs(self, size=1, random_state=None):
         return self._dist.rvs(self.alpha, size, random_state)
 
 
+class multivariate_beta_gen(dirichlet_gen):
+    r"""A multivariate beta (Dirichlet) random variable.
+
+    The ``alpha`` keyword specifies the concentration parameters of the
+    distribution.
+
+    .. versionadded:: 1.9.0
+
+    Methods
+    -------
+    pdf(x, alpha)
+        Probability density function.
+    logpdf(x, alpha)
+        Log of the probability density function.
+    rvs(alpha, size=1, random_state=None)
+        Draw random samples from a multivariate beta distribution.
+    mean(alpha)
+        The mean of the multivariate beta distribution
+    var(alpha)
+        The variance of the multivariate beta distribution
+    entropy(alpha)
+        Compute the differential entropy of the multivariate beta distribution.
+
+    Parameters
+    ----------
+    %(_dirichlet_doc_default_callparams)s
+    %(_doc_random_state)s
+
+    See Also
+    --------
+    dirichlet
+
+    Notes
+    -----
+    Each :math:`\alpha` entry must be positive. The distribution has only
+    support on the simplex defined by
+
+    .. math::
+        \sum_{i=1}^{K} x_i = 1
+
+    where :math:`0 < x_i < 1`.
+
+    If the quantiles don't lie within the simplex, a ValueError is raised.
```

Comment:
```suggestion
    If the quantiles don't lie within the simplex, a `ValueError` is raised.
```

I think we link Python docs so adding a backtick around Python keywords should link its documentation.

Reply:
Sure. I don't think I was fixing the documentation here. There are still a few issues in multivariate documentation : )